extensions/raycast/csv_to_json.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title CSV to JSON
+# @raycast.mode fullOutput
+
+# Optional parameters:
```

Comment:
You don't need to use paramaters. This function should take what's in your clipboard, modify it and update your keyboard